src/main/java/online/carsharing/entity/Car.java

```diff
@@ -0,0 +1,38 @@
+package online.carsharing.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SoftDelete;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+@Entity
+@SoftDelete
+@Table(name = "cars")
+public class Car {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private String model;
+    @Column(nullable = false)
+    private String brand;
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private Type type;
+    @Column(nullable = false)
+    private int inventory;
+    @Column(nullable = false)
+    private BigDecimal dailyFee;
+}
```

Comment:
Add empty lines between variables
```suggestion
public class Car {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    
    @Column(nullable = false)
    private String model;
    
    @Column(nullable = false)
    private String brand;
    
    @Enumerated(EnumType.STRING)
    @Column(nullable = false)
    private Type type;
    
    @Column(nullable = false)
    private int inventory;
    
    @Column(nullable = false)
    private BigDecimal dailyFee;
}
```